mutation-testing/output-pr-mutants/action.yml

```diff
@@ -0,0 +1,153 @@
+name: Output Mutants
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Download artifacts
+      uses: actions/download-artifact@v3
+
+    - name: Append output from shards
+      shell: bash
+      run: |
+        folders=("mutants-shard-big--1" "mutants-shard-big-0" "mutants-shard-big-1" "mutants-shard-big-2" "mutants-shard-big-3" "mutants-shard-big-4" "mutants-shard-big-5" "mutants-shard-big-6" "mutants-shard-big-7" "mutants-shard-small--1" "mutants-shard-small-0" "mutants-shard-small-1" "mutants-shard-small-2" "mutants-shard-small-3")
```

Comment:
i think `mutants-shard-big--1` should be `mutants-shard-big-1`?

are these shard folders based on the matrix from the calling job?

Reply:
Yes, the shard folders are collected from the composite jobs. The 'mutants-shard-big--1' is the one collected from the job without matrix, since the default check i made is if the shard is '-1' then run without shards (hence '--1'). Same applies for the small shards.